src/static-analysis/getPackageStats.js

```diff
@@ -134,9 +140,9 @@ function getPackageStats(repoPath, repoName, url) {
         };
 
       //collect row for output object
-       if(pp || pq || prcve ||prc || prce || prcon || prcore || pricon || predit || prlogv || pnext || prstyle || prtable || prtoken || prtop || pruf || prvirt || pf || pfr || pfre != null){
+       if(pp || pq || prcve ||prc || prce || prcon || prcore || pricon || predit || prlogv || prstyle || prtable || prtoken || prtop || pruf || prvirt || pf || pfr || prchat || prdata || prcomp || pfre != null){
```

Comment:
Is it ok that you insert the new items in a place that is different from where you removed `pnext` ?

It is only with in a list of or-ed together items, but for consistency moving those to the other place may be better.